Documentation/install/upgrade.rst

```diff
@@ -211,8 +211,11 @@ configuration options for each minor version.
 
   .. group-tab:: Helm
 
-    Keeping an existing `ConfigMap` with ``helm upgrade`` is currently not
-    supported.
+    .. parsed-literal::
+
+      helm upgrade |CHART_RELEASE| \\
+        --namespace kube-system cilium \\
+        --set global.configMap="$(kubectl get -n kube-system cm cilium-config -o yaml)"
```

Comment:
Should we at least split the kubectl command step out to a separate command so at least if it fails, we don't pass garbage into the configmap during upgrade?

Reply:
(also perhaps a bit easier to highlight to the user that they must set the namespace correctly)

Reply:
Thanks for that proposal Joe, I wanted to do this with a single command and at the same time wanted to make sure we should fail if `kubectl get -n kube-system cm cilium-config -o yaml` fails but I didn't remember to use 2 commands for this :facepalm: